docs/docs/administration/08-advanced/05-provisioning.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Provisioning
+description: Instructions for provisioning new PILOS instances
+---
+
+## Provisioning via JSON files
+
+PILOS provides a command to provision servers, server pools, room types, roles, users and settings with JSON files.
+
+**Example**
+
+```bash
+docker compose exec app pilos-cli provision:all provisioning_data.json
```

Comment:
The file path must be inside the container! Eg, place it into a mounted folder or copy file into container and delete afterwards

Reply:
Done.